src/Windows/Avalonia.Win32/IconImpl.cs

```diff
@@ -1,38 +1,24 @@
 ﻿using System;
 using System.Drawing;
-using System.Drawing.Imaging;
 using System.IO;
 using Avalonia.Platform;
 
 namespace Avalonia.Win32
 {
     class IconImpl : IWindowIconImpl
     {
-        private Bitmap bitmap;
-        private Icon icon;
-
-        public IconImpl(Bitmap bitmap)
-        {
-            this.bitmap = bitmap;
-        }
+        private readonly Icon icon;
 
         public IconImpl(Icon icon)
         {
             this.icon = icon;
         }
 
-        public IntPtr HIcon => icon?.Handle ?? bitmap.GetHicon();
```

Comment:
Previously GetHicon was called on each tray icon update. Apparently, Windows is not friendly to frequent gdi calls like this. And icon handle wasn't disposed after used.